modules/activiti-engine/src/test/java/org/activiti/engine/test/bpmn/event/signal/ThrowSignalEventSessionManagerTest.java

```diff
@@ -0,0 +1,58 @@
+package org.activiti.engine.test.bpmn.event.signal;
+
+import org.activiti.engine.impl.test.PluggableActivitiTestCase;
+import org.activiti.engine.test.Deployment;
+
+public class ThrowSignalEventSessionManagerTest extends PluggableActivitiTestCase {
+
+  @Deployment
+  public void testSignalRaceConditionsAfterParallelGateway() {
+
+    runtimeService.startProcessInstanceByKey("testSignalRaceConditionsAfterParallelGateway");
+    
+    // No tasks should be open then and process should have ended
+    assertEquals(0, taskService.createTaskQuery().count());

```

Comment:
@igdianov I'm sorry that I cannot understand the intention of checking task count.
Because there is no user task setting in the process definition of the test material.

Reply:
Corrected